x/gov/handler.go

```diff
@@ -65,6 +65,9 @@ func handleMsgDeposit(ctx sdk.Context, k keeper.Keeper, msg gov.MsgDeposit) (*sd
 }
 
 func handleMsgSubmitProposal(ctx sdk.Context, k keeper.Keeper, msg gov.MsgSubmitProposal) (*sdk.Result, error) {
+	if _, ok := msg.Content.(shield.ClaimProposal); ok && ctx.BlockHeight() < common.Update1Height {
```

Comment:
shouldn't it be !ok

Reply:
If the proposal is a claim proposal and the height does not reach the update height, then we return error.